types/ember__object/test/core.ts

```diff
@@ -0,0 +1,198 @@
+import Ember from 'ember';
+import { assertType } from './lib/assert';
+
+/** Newable tests */
+const co1 = new Ember.CoreObject();
+
+// TODO: Enable in TS 3.0  see: https://github.com/typed-ember/ember-cli-typescript/issues/291
```

Comment:
Enable this after https://github.com/typed-ember/ember-cli-typescript/issues/291 is resolved